packages/stellar-axelar-std-derive/src/storage.rs

```diff
@@ -110,6 +128,106 @@ impl Value {
         }
     }
 
+    pub fn status_fns(
```

Comment:
```suggestion
    fn status_fns(
```
does it need to be `pub`?